components/CommonCompStreak.js

```diff
@@ -3,8 +3,17 @@ import { StyleSheet, Button, View, SafeAreaView, Text, Alert } from 'react-nativ
 
 const CommonCompStreak = (prop) => {
   return (
-    <button>{prop.numOfStreaks}</button>
```

Comment:
I have made a streak component and the style in this PR: https://github.com/394-s21/habit-tracker/pull/11. So the button is no longer needed here :(